blockchain/blockdao.go

```diff
@@ -417,14 +360,41 @@ func (dao *blockDAO) getBlockchainHeight() (uint64, error) {
 
 // getTotalActions returns the total number of actions
 func (dao *blockDAO) getTotalActions() (uint64, error) {
-	value, err := dao.kvstore.Get(blockNS, totalActionsKey)
+	indexer, err := dao.kvstore.CountingIndex(totalActionBucket)
```

Comment:
totalAction need to calculate separately,sometimes getTotalActions will be called without '-plugin=gateway' enabled